File: Apps/app1Sorteio/app/src/main/java/com/cursoandroid/app1sorteio/MainActivity.java
package com.cursoandroid.app1sorteio;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void sorteiaNumero(View view){

        TextView txtSorteio = findViewById(R.id.numeroSorteado);

        int numeroAleatorio = new Random().nextInt(5);

        txtSorteio.setText("Numero sorteado foi: " + numeroAleatorio);
    }
}

File: Kotlin/app/src/main/java/com/cursoandroid/kotlin/MainActivity.kt
package com.cursoandroid.kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.TextureView
import android.view.View
import android.widget.TextView
import kotlin.random.Random

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun sortearNumero(view: View){

        var texto = findViewById<TextView>(R.id.resultado)
        var numeroSorteado = Random.nextInt(10);

        texto.setText("Numero sorteado e: " + numeroSorteado)
        //or
        texto.text = "Numero sorteado e: " + numeroSorteado

    }

}